src/cli_plugin/index.js

```diff
@@ -0,0 +1,5 @@
+// load the babel options seperately so that they can modify the process.env
+// before calling babel/register
+const babelOptions = require('../optimize/babelOptions').node;
```

Comment:
AFAIK, these need to be require statements, because this is where babel is being wired up.